frontend/src/components/forms/CreateLog.tsx

```diff
@@ -207,6 +205,15 @@ const CreateLog = ({ getRecords, countRecords, setUserPageNum }: Props) => {
 
   // fetch resident + employee data for log creation
   const getLogEntryOptions = async () => {
+    const buildingsData = await BuildingAPIClient.getBuildings();
+
+    if (buildingsData && buildingsData.buildings.length !== 0) {
+      const buildingLabels: BuildingLabel[] = buildingsData.buildings.map(
+        (building) => ({ label: building.name!, value: building.id! }),
```

Comment:
Does the `address` property for the building ever get used?

Reply:
no, but i figured it was better to not have the backend return a list of `BuildingLabel` objects. My reasoning was that if we ever wanted to access the address for whatever reason, we wouldn't have to make a new endpoint in the backend. I also figured that we would want to use the address eventually because why else would it be there